WordGame/ViewController.swift

```diff
@@ -8,13 +8,23 @@
 import UIKit
 
 class ViewController: UITableViewController {
-    var allWords = [String]()
-    var usedWords = [String]()
+    var allWords: [String] = []
+    var usedWords: [String] = []
+    let defaults = UserDefaults.standard
```

Comment:
Just a note that we usually use `UserDefaults` to store user's preferences - like favorites, user id etc. For cases like this game, we'd probably use another strategy to persist data, like CoreData, but it's valid as an exercise.